crux-bench/time-series-bench/src/time_series_bench/main.clj

```diff
@@ -0,0 +1,20 @@
+(ns time-series-bench.main
+  (:require [time-series-bench.ts-weather :as weather]
+            [time-series-bench.ts-devices :as devices]
+            [clojure.data.json :as json]))
+
+(defn -main []
+  (let [start-time (System/currentTimeMillis)]
+    (println (json/write-str {:crux.bench/bench-type ::main
```

Comment:
I'd go with normal `log/info` here, already has namespace and time info - we're not planning to use these for metrics?

Reply:
With Cloudwatch, you need to have json output on its own. If you use `log/info` it outputs INFO TIMESTAMP - <json> and cloudwatch then won't parse the output

Reply:
Although in this case, as these are basically logging messages rather than metrics we plan to use (unless we did want to use `::time-taken` as a final metric), Log/info would be appropriate.

Reply:
I'd say the json format is just more rich/convenient in general. This way we can do `level="info"` rather than using `like` which just searches for a string, or `parse` where we'd have to input the logging format manually. Using the json format means that cloudwatch does all the parsing for us.